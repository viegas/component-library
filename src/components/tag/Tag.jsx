import React from 'react'
import PropTypes from 'prop-types'

import * as Styled from './Tag.styles'

const Tag = ({text}) => {
    return (
        <Styled.Wrapper>
            {text}
        </Styled.Wrapper>
    )
}

Tag.propTypes = {
    text: PropTypes.string.isRequired,
}

export default Tag;
