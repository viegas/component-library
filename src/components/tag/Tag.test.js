import Tag from './Tag';
import * as Styled from './Tag.styles';

describe('<Tag />', () => {
    it('Should render correctly', () => {
        const wrapper = shallow(<Tag text={'Some text'} />);
        expect(wrapper).toMatchSnapshot();
    });

    styledComponentsSnapshots(Styled, '*');
});
