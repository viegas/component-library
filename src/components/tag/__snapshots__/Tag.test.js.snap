// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> <Styled.Wrapper /> Should render correctly 1`] = `
.c0 {
  cursor: default;
  display: inline-block;
  font-size: 12px;
  color: #39739d;
  background-color: #e1ecf4;
  padding: 5px 7px;
  border-width: 1px;
  border-style: solid;
  border-radius: 3px;
  border-color: transparent;
}

.c0:hover {
  color: #2c5777;
  background-color: #d1e5f1;
}

<div
  className="c0"
/>
`;

exports[`<Tag /> Should render correctly 1`] = `
<Tagstyles__Wrapper>
  Some text
</Tagstyles__Wrapper>
`;
